gempy/assets/model_compare.py

```diff
@@ -0,0 +1,136 @@
+import sys
+import numpy as np
+import matplotlib.pyplot as plt
+sys.path.append("/Users/zhouji/Documents/github/gempy")
+
```

Comment:
@Leguark  This file is where I test the tensorflow_graph.py by passing the information from the model's interpolator to TFgraph, hope this could give you some clue how to integrate this into gempy